Add `The Gates Open for the Lazer Grand Arena 2026` news post, `LGA 2026` wiki article (#14681)
ppy/osu-wiki: news/2026/2026-05-02-the-gates-open-for-the-lazer-grand-arena-2026.md

---
layout: post
title: The Gates Open for the Lazer Grand Arena 2026
date: 2026-05-02 20:00:00 +0000
series: world_cups
---

Find out how to participate and where to watch!

![](/wiki/shared/news/2026-05-02-the-gates-open-for-the-lazer-grand-arena-2026/banner.jpg)

> For this year, LGA returns with its core format unchanged, except for the addition of a bracket reset in the Grand Finals if required. We think the foundation has matured enough for us to focus on bettering the player and viewer experiences.

Reviewer: ```suggestion
For this year, LGA returns with its core format unchanged, except for the addition of a bracket reset in the Grand Finals if required. We think the tournament's foundation has matured enough for us to focus on bettering the player and viewer experiences.
```

> For more details pertaining to the tournament, please refer to the **[Lazer Grand Arena 2026 wiki article](/wiki/Tournaments/LGA/2026)**. You can also discuss this event in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/2202575)** or the public **[community server](https://discord.gg/0Vxo9AsejDkGlk3H)**.

Reviewer: ```suggestion
For more details pertaining to the tournament, please refer to the **[Lazer Grand Arena 2026 wiki article](/wiki/Tournaments/LGA/2026)**. You can also discuss this event in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/2202575)** or the `#tournaments` channel in osu!'s public **[community server](https://discord.gg/0Vxo9AsejDkGlk3H)**.
```

Author: This one links to the community osu! server, not osu!dev - #tournaments there is meant mainly for tournament committee discussions

## Qualifiers mappool showcase

Catch the full qualifier mappool reveal live on the [osulive Twitch channel](https://twitch.tv/osulive) this **Sunday, May 3 at 18:00 UTC**.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Qualifier mappool showcase | May 3 (18:00 UTC) |
| Qualifier stage | May 3 (19:00 UTC) – May 10 (19:00 UTC) |
| Qualifiers results / Week 1 mappool showcase | May 10 (21:00 UTC) |

> | Bracket stage (single elimination) | May 17 & May 18 |

Reviewer: this and the next line conflict with the dates on the schedule on the wiki
(i assume the wiki is correct though. news article dates are sunday/monday for bracket stage and top 8)

Author: Yeah, these are definitely incorrect. Addressed in [a5bd745](https://github.com/ppy/osu-wiki/pull/14681/commits/a5bd745ec5df3d1d79c02b584fc337a7488a0145)

For this stage, your **lowest scoring map will be removed for the seeding calculations**, and **NoFail is enforced** for every map. Only the **highest score for each map is counted**. To have your scores considered for seeding, you need to **have at least one submitted play for each map of the playlist** (indicated by the blue checkmark with a "Completed Beatmaps" tooltip inside the playlist). Refer to the following image for an example of a valid submission:

![Player example for qualifiers playlist](/wiki/shared/news/2026-05-02-the-gates-open-for-the-lazer-grand-arena-2026/azer-steamhappy.png)

Afterwards, we will apply [%MAX seeding](/wiki/Tournaments/Common_seeding_methods#percent-max) to all valid entries. The **top 32** players will be invited to play in the bracket stage.

## Bracket stage match format

Here's a refresher:

![](/wiki/shared/news/2026-05-02-the-gates-open-for-the-lazer-grand-arena-2026/bracket-stage.jpg)

> In the Lazer Grand Arena format, the match starts with the Blue player (low seed). Both players alternate to ban one map, protect one map, and a second ban. Protected maps can't be banned by either player and may only be picked by the player who protected them.

Reviewer: ```suggestion
In the Lazer Grand Arena format, the match starts with the Blue player (low seed). Both players alternate to ban one map, protect one map, and ban a second map. Protected maps can't be banned by either player and may only be picked by the player who protected them.
```

The match itself is a best-of-five, with each point being played in "sets" of two maps each. The Red player (high seed) makes the first pick, following an A-B-B-A order for the remainder of the match. In practice, this means that players take turns first picking for each new set.

Scoring is cumulative within a set. The scores from the two maps are combined, and the player with the highest total score earns one point. The first player to reach three points wins. If the match reaches a 2 to 2 tie, the fifth set expands into a three map brawl, concluding with a brand new osu! original beatmap.

> The bracket stage kicks off with **single elimination** for the Round of 32 and Round of 16. Once we get to the Top 8, a Losers bracket is introducted, shifting the format to a **double elimination bracket** for the remainter of the tournament.

Reviewer: ```suggestion
The bracket stage kicks off with **single elimination** for the Round of 32 and Round of 16. Once we get to the Top 8, a Losers bracket is introduced, shifting the format to a **double elimination bracket** for the remainder of the tournament.
```
